examples/snippets/dataset/manage/add_complex_attribute.py

```diff
@@ -0,0 +1,38 @@
+"""
+An example script to change dataset attributes for a Tamr dataset
+"""
+import tamr_toolbox as tbox
+
+
+# load example multi config
+my_config = tbox.utils.config.from_yaml("examples/resources/conf/migrate_dataset.config.yaml")
+
+# Use the configuration to create a global logger
+LOGGER = tbox.utils.logger.create(__name__, log_directory=my_config["logging_dir"])
+
+# Create the source tamr client
+client = tbox.utils.client.create(**my_config["source_migration_instance"])
+
+
+dataset_name = "<your dataset name>"
+# Get dataset from Tamr instance
+dataset = client.datasets.by_name(dataset_name)
```

Comment:
There's no need to create a LOGGER here.  For snippets we should try to be as concise as possible.
```suggestion
# Create the source tamr client
client = tbox.utils.client.create(**my_config["source_migration_instance"])

# Get dataset from Tamr instance
dataset_name = "<your dataset name>"
dataset = client.datasets.by_name(dataset_name)
```